src/Moq/Language/Flow/SetupPhrase.cs

```diff
@@ -162,5 +162,10 @@ public void Verifiable(string failMessage)
 		{
 			this.setup.Verifiable(failMessage);
 		}
+
+		public override string ToString()
+		{
+			return setup.Expression.ToStringFixed();
```

Comment:
Ideally, this would just read `return this.setup.ToString()` in order to avoid duplication of logic that belongs in the setup. But I realise this isn't currently possible, because e.g. `MethodCall` includes a failure message in its `ToString()` formatting.

This should be refactored at some later time, but how you've done it is fine for now. :+1: